kie-camel/src/test/java/org/kie/camel/embedded/camel/component/cxf/CxfSoapTest.java

```diff
@@ -21,6 +21,7 @@
 import javax.xml.namespace.QName;
 import javax.xml.soap.MessageFactory;
 import javax.xml.soap.SOAPBody;
+import javax.xml.soap.SOAPConstants;
```

Comment:
This import seem to be redundant.

Reply:
True story. Fixed and also cleaned up other unused lines.